src/getCoinCombination.test.js

```diff
@@ -7,4 +7,32 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+  test(`returns correct combination for 1 cent`, () => {
```

Comment:
You have splitted two different ways, how to make tests. Pay attention on 6-8 lines. In this task - You need to use this structure 
```
  it(`should return array with '0' when '0' is entered`, () => {
    expect(getCoinCombination(0))
      .toEqual([0, 0, 0, 0]);
  });
```